PWGGA/GammaConv/AliAnalysisTaskGammaConvV1.cxx

```diff
@@ -3780,33 +3930,33 @@ void AliAnalysisTaskGammaConvV1::CalculatePi0Candidates(){
   
   // Conversion Gammas
   if(fGammaCandidates->GetEntries()>1){
-    for(Int_t firstGammaIndex=0;firstGammaIndex<fGammaCandidates->GetEntries()-1;firstGammaIndex++){
-      AliAODConversionPhoton *gamma0=dynamic_cast<AliAODConversionPhoton*>(fGammaCandidates->At(firstGammaIndex));
-      if (gamma0==NULL) continue;
-      for(Int_t secondGammaIndex=firstGammaIndex+1;secondGammaIndex<fGammaCandidates->GetEntries();secondGammaIndex++){
-        AliAODConversionPhoton *gamma1=dynamic_cast<AliAODConversionPhoton*>(fGammaCandidates->At(secondGammaIndex));
-        //Check for same Electron ID
-        if (gamma1==NULL) continue;
-        if(gamma0->GetTrackLabelPositive() == gamma1->GetTrackLabelPositive() ||
-        gamma0->GetTrackLabelNegative() == gamma1->GetTrackLabelNegative() ||
-        gamma0->GetTrackLabelNegative() == gamma1->GetTrackLabelPositive() ||
-        gamma0->GetTrackLabelPositive() == gamma1->GetTrackLabelNegative() ) continue;
-
-        AliAODConversionMother *pi0cand = new AliAODConversionMother(gamma0,gamma1);
-        pi0cand->SetLabels(firstGammaIndex,secondGammaIndex);
-        pi0cand->CalculateDistanceOfClossetApproachToPrimVtx(fInputEvent->GetPrimaryVertex());
-        
-        AliVTrack * ENegTrack_Gamma0 = fiPhotonCut->GetTrack(fInputEvent, gamma0->GetTrackLabelNegative());
-        AliVTrack * EPosTrack_Gamma0 = fiPhotonCut->GetTrack(fInputEvent, gamma0->GetTrackLabelPositive());
+    if (fDoTreeForMesonML){
```

Comment:
Have you checked if the normal analysis is still working? The if(fDoTreeForMesonML) makes me wonder if the pi0 calculation is now only enabled for the ML part